packages/style-engine/class-wp-style-engine.php

```diff
@@ -404,7 +444,10 @@ public function generate( $block_styles, $options ) {
 		if ( ! empty( $css_rules ) ) {
 			// Generate inline style rules.
 			foreach ( $css_rules as $rule => $value ) {
-				$filtered_css = esc_html( safecss_filter_attr( "{$rule}: {$value}" ) );
+				// $filtered_css = esc_html( safecss_filter_attr( "{$rule}: {$value}" ) );
+				// @TODO disabling escaping only for this test.
+				// The `transition` property is filtered out otherwise.
+				$filtered_css = "{$rule}: {$value}";
```

Comment:
Let's avoid this and instead filter the rules defined in Core:

```
add_filter(
	'safe_style_css',
	function( $safe_rules ) {
		$safe_rules[] = 'transition';
		return $safe_rules;
	}
);
```

Reply:
Nice work with the filter! Looks like a sound approach. 

Thanks! 🙇